HelpSource/Classes/Decay.schelp

```diff
@@ -19,18 +16,78 @@ argument::in
 The input signal.
 
 argument::decayTime
-60 dB decay time in seconds.
+The time it takes for the input signal to decay 60 dB.
 
 argument::mul
+The output will be multiplied by this value.
 
 argument::add
+This value will be added to the output.
 
 Examples::
 
 code::
-plot({ Decay.ar(Impulse.ar(1), 0.01) });
+// Plot the exponentially decaying envelope
+(
+var t60 = 0.1; // decayTime
+plot({ Decay.ar(Impulse.ar(0), t60) }, t60);
+)
+::
+
+code::
+// Used as an evelope generator
+(
+play({
+    Decay.ar(
+        in: Impulse.ar(XLine.kr(5,50,10), 0.25),
+        decayTime: 0.2,
+        mul: FSinOsc.ar(300)
+    )
+})
+)
+::
+
+As you might hear in the previous example, when code::Decay:: is used as a modulator, its immediate onset can lead to discontinuities and audible clicks. You can see this in the waveform:
```

Comment:
In the helpfiles (at least in the reference, not in the tutorial), there is a tendency not to address the reader directly. It is a matter of choisce of course! I always avoid it, because I find it a little distracting here. (I just wanted to mention this)

Reply:
@telephon, are you suggesting the use of "we" rather "you" here?

If yes, this is a small detail, and does meet my preferences... but brings up a larger question regarding documentation style.

ATM, the documentation [Style Guidelines](https://github.com/supercollider/supercollider/wiki/Style-Guidelines%3A-SCDocs) don't include a discussion around language and tone.

I'd be up for pressing go here.... but if @mtmccrea is up for changing the voice, I'd be fine with that, too.

(@mtmccrea thanks for all your work on this!!)

Reply:
I've updated the language to the passive voice now.

I try to keep track of ideas for schelp style guidelines, so I'll add a note about the voice used as a point for discussion.  One day it may turn into a proposal, but there's a lot to discuss around docs, in some ways more than code style guidelines.

Reply:
> @telephon, are you suggesting the use of "we" rather "you" here?
> 
> If yes, this is a small detail, and does meet my preferences... but brings up a larger question regarding documentation style.

No actually, I wasn't thinking of "we" instead of "you", but sometimes an aspect of passive voice, sometimes giving the voice to the entities we talk about:

Instead of writing:
You can modulate the frequency at audio rate

I would write
The frequency can be modulated at audio rate

Instead of writing:
You should never have a loop without a wait, it will crash

I would write
A loop without a wait will crash

Instead of writing:
Be careful not to forget this argument, otherwise ...

I would write
This argument should always be given, otherwise ...

Instead of writing:
You can equally do A, B, and C

I would write
A, B, and C are equally possible